src/styles/scss/App.scss

```diff
@@ -0,0 +1,51 @@
+.App {
+  text-align: center;
+  display: flex;
+  width: 700px;
+  flex-direction: column;
+  margin: 0 auto;
+
+  .display,
+  .btn-panel {
+    display: flex;
+    width: 100%;
+  }
+
+  .btn-panel {
+    flex-direction: column;
+
+    .btn {
+      width: 25%;
+      height: 100%;
+      border: 1px solid black;
+      font-size: large;
+    }
+
+    .btn-group:last-child .btn:first-child {
+      width: 50%;
+    }
```

Comment:
- [ ] This should be set using `props`, not directly in the CSS.

Reply:
logic moved to the component